ard/collection/optiwindnet_wrap.py

```diff
@@ -22,13 +25,37 @@ def _own_L_from_inputs(inputs: dict, discrete_inputs: dict) -> nx.Graph:
     VertexC[:T, 1] = inputs["y_turbines"]
     VertexC[-R:, 0] = inputs["x_substations"]
     VertexC[-R:, 1] = inputs["y_substations"]
+
+    # add jitter to duplicate turbine/substation positions
+    VertexCTR = np.vstack([VertexC[:T, :], VertexC[-R:, :]])
+    jitter_eps = 1.0e-3  # 0.001m magnitude jitter
+    jitter_normal = np.array([-1.0, 1.0])  # on a fixed axis we'll normalize
+    jitter_normal = jitter_normal / np.sqrt(np.sum(jitter_normal**2))
+    repeat_accumulate = np.array(
```

Comment:
More descriptive comments here would be helpful. I think you are counting how many vertices there are in the array that match each given vertex